test/aircraft/Pilot/Pilot.spec.js

```diff
@@ -855,7 +855,7 @@ ava('.initiateHoldingPattern() returns correct readback when hold implemented su
     t.deepEqual(result, expectedResult);
 });
 
-ava('.maintainAltitude() returns early responding that they are unable to maintain the requested altitude', (t) => {
+ava.skip('.maintainAltitude() returns early responding that they are unable to maintain the requested altitude', (t) => {
```

Comment:
Should this be skipped?

Reply:
Yeah, both of these tests are failing after the upgrades.  If I remember correctly, it's just a precision thing due to a newer version of node, ava, etc and I haven't had time to go back and address it.

Reply:
this is passing now, just removed the `.skip()`